Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -2403,327 +2403,430 @@ public static void RemoveAssetFromHeroLoadout(string assetID, string heroID, Act
-        public static void ListPlayerCharacters(Action<LootLockerPlayerCharacterListResponse> onComplete)
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerPlayerClassListResponse</param>
+        public static void ListPlayerClasses(Action<LootLockerPlayerClassListResponse> onComplete)
         {
             if (!CheckInitialized())
             {
-                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerPlayerCharacterListResponse>());
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerPlayerClassListResponse>());
                 return;
             }
-            LootLockerAPIManager.ListPlayerCharacters(onComplete);
+            LootLockerAPIManager.ListPlayerClasses(onComplete);
+        }
+
+        [Obsolete("This function is deprecated and will be removed soon. Please use the function GetClassLoadout() instead")]
+        public static void GetCharacterLoadout(Action<LootLockerClassLoadoutResponse> onComplete)
+        {
+            GetClassLoadout(onComplete);
         }
 
         /// <summary>
-        /// Get all character loadouts for your game.
+        /// Get all class loadouts for your game.
         /// </summary>
-        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerCharacterLoadoutResponse</param>
-        public static void GetCharacterLoadout(Action<LootLockerCharacterLoadoutResponse> onComplete)
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerClassLoadoutResponse</param>
+        public static void GetClassLoadout(Action<LootLockerClassLoadoutResponse> onComplete)
         {
             if (!CheckInitialized())
             {
-                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerCharacterLoadoutResponse>());
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerClassLoadoutResponse>());
                 return;
             }
-            LootLockerAPIManager.GetCharacterLoadout(onComplete);
+            LootLockerAPIManager.GetClassLoadout(onComplete);
+
         }
 
-        /// <summary>
-        /// Get a character loadout from a specific player on the current platform.
-        /// </summary>
-        /// <param name="player_id">ID of the player</param>
-        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerCharacterLoadoutResponse</param>
-        public static void GetOtherPlayersCharacterLoadout(string player_id, Action<LootLockerCharacterLoadoutResponse> onComplete)
+        [Obsolete("This function is deprecated and will be removed soon. Please use the function GetOtherPlayersClassLoadout() instead")]
+        public static void GetOtherPlayersCharacterLoadout(string player_id, Action<LootLockerClassLoadoutResponse> onComplete)
```

Comment:
You seem to have forgotten to re-create this overload.

Reply:
I've added it now thank you

Reply:
I can't see the commit, did you forget to push?